packages/react-vapor/src/components/tab/Tab.tsx

```diff
@@ -34,6 +35,7 @@ export class Tab extends React.Component<ITabProps, any> {
 
         return (
             <div className={className} onClick={this.handleSelect}>
+                {!!this.props.children && this.props.children}

```

Comment:
You can simply use {this.props.children} since the default value is undefined which doesn't get rendered